subsys/bluetooth/host/classic/sdp.c

```diff
@@ -1678,11 +1702,6 @@ int bt_sdp_register_service(struct bt_sdp_record *service)
 		return -EEXIST;
 	}
 
-	if (num_services == BT_SDP_MAX_SERVICES) {
```

Comment:
Could we merge these change into https://github.com/zephyrproject-rtos/zephyr/pull/94543/commits/448d5b85bb77398f81f78118bf6013d4c792b960？